Formula/qshell.rb

```diff
@@ -0,0 +1,14 @@
+class Qshell < Formula
+  desc "Qiniu Cloud Resource Management Tool "
+  homepage "https://github.com/qiniu/qshell"
+  url "https://dn-devtools.qbox.me/qshell-2.1.3.tar.gz"
```

Comment:
if i want to publish a binary executable file, is there any docs i can refer to? many thanks